### PR TITLE
Update/docs improve and fixes

### PR DESCRIPTION
**Pull request by ka1bi4**
- **Docs update.**
- **Improving documentation as part of the hacktorberfest festival ;)**
- **Improving the readability of the text and correcting minor typos.**

### CODE REVIEW

**Comment by ka1bi4** [comment]
Hi,
Improving documentation as part of the hacktorberfest festival ;)

File: docs/pages/Deploying/Quickstart.md
Comment (dartpain) [inline docs/pages/Deploying/Quickstart.md:1]: Please add that its bash here too please

**Review by dartpain — CHANGES_REQUESTED** [review]
Just a minor change

File: docs/pages/Deploying/Quickstart.md
Comment (ka1bi4) [reply docs/pages/Deploying/Quickstart.md:1]: Yeah! I also fixed some . in the end of commands and make links localhost

**Comment by ka1bi4** [comment]
@dartpain, thanks!
Done it

**Review by dartpain — APPROVED** [review]

**Comment by dartpain** [comment]
Thank you!

**Pull request merged**